eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         displayName: "Publish Results ($(TestTargetFramework))"
         inputs:
           testResultsFiles: "**/*.trx"
-          testRunTitle: "$(OSName) $(TestTargetFramework)"
+          testRunTitle: "${{ parameters.ScenarioName }} $(OSName) $(TestTargetFramework)"
```

Comment:
Given that ScenarioName isn't for most things in this repo this now means all the run titles are going start with an empty space. Maybe we need to set some sort of reasonable default name or maybe the run title correctly gets trimmed and if not perhaps we can fix that be removing the quotes around the title.  @danieljurek do you know if we have this any other places?

Reply:
I think we should something like this for future proofing. 

The problem we were seeing here is that the name of the cloud was not present in the lists of failed tests and one would have to dig deeper to infer the cloud being tested. This is a temporary measure in a branch that is not intended to merge to `master` so that we can properly categorize which test failures are happening in which cloud.